gensim/models/translation_matrix.py

```diff
@@ -139,29 +137,35 @@ def __init__(self, word_pair, source_lang_vec, target_lang_vec, random_state=Non
         self.source_space = None
         self.target_space = None
 
-    def train(self, word_pair):
+        if word_pairs is not None:
+            if len(word_pairs[0]) != 2:
+                raise ValueError("Each training data item must contain two different language words.")
+            self.train(word_pairs)
+
+    def train(self, word_pairs):
         """
-        build the translation matrix that mapping from source space to target space.
+        Build the translation matrix that mapping from source space to target space.
 
         Args:
-            `word_pair` (list): a list pair of words
+            `word_pairs` (list): a list pair of words
             `source_space` (Space object): source language space
             `target_space` (Space object): target language space
 
         Returns:
             `translation matrix` that mapping from the source language to target language
         """
-        self.source_space = Space.build(self.source_lang_vec, set(self.source_word))
-        self.target_space = Space.build(self.target_lang_vec, set(self.target_word))
+        self.source_word, self.target_word = zip(*word_pairs)
+        if self.translation_matrix is None:
```

Comment:
But if I called train twice, in the second time I don't fit model.
Please remove this `if`